src/components/common/TechnologyLogo/styles.js

```diff
@@ -0,0 +1,32 @@
+export default theme => ({
+  root: {
+    width: '50%',
```

Comment:
The component should have an independent layout. So you can add the layout outside as the way you like it. So this component should not have a defined width or it should be inline-block with width auto or display block.